frontend/src/components/Election/Admin/AdminHome.tsx

```diff
@@ -341,6 +374,20 @@ const AdminHome = ({ election, permissions, fetchElection }: Props) => {
             console.log(err)
         }
     }
+
+    const archiveElection = async () => {
+        console.log("archiving election")
+        if (window.confirm('Are you sure you wish to archive this election? This action cannot be undone')){
```

Comment:
nit

Using the default popup is fine for now, but we should probably implement our own in the future

Maybe I can iterate on my popup in [this pr](https://github.com/Equal-Vote/star-server/pull/273) to make it a reusable component

Reply:
Yea I looked at some options for a pop up but it was more than I wanted for this PR. Having a reusable component would be nice.